src/services/hubApi.ts

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+import * as queryString from "query-string";
+import { FilterParamsObject } from "../components/Hub/HubFilters";
+
+const HUB_API_ENDPOINT = process.env.REACT_APP_HUB_API!;
+
+export const getHubImages = async (filters: FilterParamsObject) => {
+  const response = await axios.get(HUB_API_ENDPOINT + "/images", {
+    params: filters,
```

Comment:
We should probably have some sort of `status` field to handle errors by other components in case there is an issue getting images